README.md

```diff
@@ -110,39 +110,40 @@ Doing this causes the NATS Streaming server to track the last acknowledged messa
 ```javascript
 var stan = require('node-nats-streaming').connect('test-cluster', 'client-123');
```

Comment:
maybe these examples should also be here? https://github.com/nats-io/node-nats-streaming/tree/master/examples/readme_samples /cc @aricart